chart/kubeapps/templates/frontend/configmap.yaml

```diff
@@ -65,6 +65,16 @@ data:
       # used with an auth-proxy where k8s credentials never leave
       # the cluster). See clusters option.
     {{- range .Values.clusters }}
+      
+    {{- if not $.Values.featureFlags.operators }}
+
+      # Operators support disabled
+      location ~* /api/clusters/{{ .name }}/.*operators.coreos.com.* {
```

Comment:
With this, should be enough for Nginx to avoid calls to URLs related to operators mgmt.

Reply:
Exactly, this piece should be temporary until we get rid of the direct calls to k8s API.